src/api_server/swagger/firecracker.yaml

```diff
@@ -604,9 +630,23 @@ definitions:
       vmm_version:
         description: MicroVM hypervisor build version.
         type: string
-      app_name:
-        description: Application name.
+
+  LoadSnapshotParams:
+    type: object
+    required:
+      - mem_file_path
+      - snapshot_path
+    properties:
+      enable_diff_snapshots:
+        type: bool
```

Comment:
```suggestion
        type: boolean
```

Reply:
I'd prefer to keep this PR to simple reordering.